packages/dependency-royalty-management/contracts/RoyaltyDistributorV2.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IERC2981Upgradeable} from "@openzeppelin/contracts-upgradeable/interfaces/IERC2981Upgradeable.sol";
+import {IRoyaltyManager} from "./interfaces/IRoyaltyManager.sol";
+
+/// @title RoyaltyDistributor
+/// @author The Sandbox
+/// @notice Contract for distributing royalties based on the ERC2981 standard.
+abstract contract RoyaltyDistributorV2 {
```

Comment:
I understand that we need another contract without Initialize but this naming is bad, i think we should either:
- rename it so we understand its an implementation without init
- or put the code directly in land (but still use royaltymanager)